### PR TITLE
Add support of repository name aliases.

### PR DESCRIPTION
**Pull request by gimite**
#3

### CODE REVIEW

**Comment by gimite** [comment]
Ping? @skywhale

File: app/utils.py [hunk 1]
```diff
@@ -870,6 +898,8 @@ def __init__(self, request, response, env):
         if not model.Repo.get_by_key_name(self.repo):
             if legacy_redirect.do_redirect(self):
                 return legacy_redirect.redirect(self)
+            if self.maybe_redirect_for_repo_alias(request):
```
Comment (skywhale) [inline app/utils.py]: maybe we want to do this before the authorization check, which would be wasted if we decide to redirect?

**Comment by skywhale** [comment]
Thanks for the change. This is going to be super useful !!

LGTM modulo one comment.

**Comment by skywhale** [comment]
And I'm really sorry for the delayed response.

File: app/utils.py (hunk 1 above)
Comment (gimite) [inline app/utils.py]: Done.

**Comment by gimite** [comment]
Thanks for the review. Pulled.

**Pull request merged**